cmd/ci-reporter/cmd/root.go

```diff
@@ -205,25 +249,26 @@ func PrintReporterData(cfg *Config, reports *CIReportDataFields) error {
 					data = append(data, []string{record.ID, record.Title, record.Category, record.Status, fmt.Sprintf("%v", record.Sigs), record.URL, record.CreatedTimestamp})
 				}
 			}
+			table.SetBorders(tablewriter.Border{Left: true, Top: false, Right: true, Bottom: false})
+			table.AppendBulk(data)
+			table.SetCenterSeparator("|")
+			table.SetAutoMergeCells(true)
+			table.Render()
 
+			// write a summary
 			countCategories := map[string]int{}
 			categoryIndex := 2
 			for i := range data {
 				countCategories[data[i][categoryIndex]]++
 			}
 			categoryCounts := ""
 			for category, categoryCount := range countCategories {
-				categoryCounts += fmt.Sprintf("%s:%d\n", category, categoryCount)
+				categoryCounts += fmt.Sprintf("%s:%d ", category, categoryCount)
 			}
-			if cfg.ShortReport {
-				table.SetFooter([]string{fmt.Sprintf("Total: %d", len(data)), "", categoryCounts, ""})
-			} else {
-				table.SetFooter([]string{fmt.Sprintf("Total: %d", len(data)), "", categoryCounts, "", "", "", ""})
+			_, err = out.WriteString(fmt.Sprintf("\nSUMMARY - Total:%d %s\n", len(data), categoryCounts))
+			if err != nil {
+				return errors.Wrap(err, "could not write to output stream")
```

Comment:
When we're dealing with a result with a single error, we usually prefer:

```suggestion
			if _, err := out.WriteString(fmt.Sprintf("\nSUMMARY - Total:%d %s\n", len(data), categoryCounts)); err != nil {
				return errors.Wrap(err, "could not write to output stream")
```